spec/mailers/renewal_reminder_mailer_spec.rb

```diff
@@ -18,4 +18,20 @@
       expect(mail.body.encoded).to include(registration.reg_identifier)
     end
   end
+
+  describe ".second_reminder_email" do
+    let(:registration) { create(:registration, expires_on: 3.days.from_now) }
+
+    before do
+      allow(Rails.configuration).to receive(:email_service_email).and_return("test@example.com")
+    end
+
+    it "sends a second reminder email" do
+      mail = described_class.first_reminder_email(registration)
```

Comment:
I'm guessing this should be

```suggestion
      mail = described_class.second_reminder_email(registration)
```

This makes me think we should have some other expectation in the tests to confirm we are seeing the right email. Perhaps

```ruby
# First reminder
expect(mail.subject).to include("registration expires soon, renew online now")

# Second reminder
expect(mail.subject).to include("Final reminder")
```

What do you think?